Assets/Scripts/Model/Cards/IType.cs

```diff
@@ -9,6 +10,6 @@ public interface IType
         bool Installable { get; }
         bool Rezzable { get; }
         List<IInstallDestination> FindInstallDestinations(Game game);
-        Stealable Stealable { get; }
+        IStealOption DefaultStealing(Card card);
```

Comment:
If we switch to C# 8, we can use default implementations in interfaces. It means that I could set default `=> new CannotSteal();` and add extra code only to the `Agenda : IType` impl.

- [ ] Go for Unity 2020 for C# 8?

Reply:
Let's go for it! 😄

Reply:
Seems that even with C# 8, [Unity doesn't support](https://docs.unity3d.com/2020.2/Documentation/Manual/CSharpCompiler.html) what I need:
> Note that the C# 8 feature “Default interface methods” is not currently supported.